tests/browser-integration/src/browserSync.test.ts

```diff
@@ -182,9 +179,6 @@ describe("Browser sync", () => {
       AccountSchema: CustomAccount,
     });
 
-    // TODO: Wait for sync doesn't work on the IndexedDB storage peer as it just waits for the content to be pushed
-    await new Promise((resolve) => setTimeout(resolve, 500));
-
```

Comment:
Improved the API and stability around storage, so we can now remove these workarounds from the browser tests.